templates/pkg/resource/references.go.tpl

```diff
@@ -117,7 +117,7 @@ func resolveReferenceFor{{ $field.FieldPathWithUnderscore }}(
 
 {{- $fp := ConstructFieldPath $field.Path -}}
 {{ $_ := $fp.Pop -}}
-{{ $isNested := gt $fp.Size 0 -}}
+{{ $isNested := gt $fp.Size 0 -}} 
```

Comment:
Mind removing the whitespace at the end of this line please, @Vandita2020? :)

Reply:
Done.